docs/source/known-issues.rst

```diff
@@ -28,6 +28,8 @@ CUDA
   
   Therefore, from version 4.5, the default behavior for Kokkos is to preventively disable `cudaMallocAsync.`
 
+- cuda 11.0 and 11.2 are not compatible with glibc 2.34 librt stubs. That issue is related to how the CMake package handles linking with librt. For more information please look at issue `#7512 <https://github.com/kokkos/kokkos/issues/7512>`_.
```

Comment:
```suggestion
- Cuda 11.0 and 11.2 are not compatible with glibc 2.34 librt stubs. That issue is related to how the CMake package handles linking with librt. For more information please look at issue `#7512 <https://github.com/kokkos/kokkos/issues/7512>`_.
```

Reply:
Downgrading `CMake` to less than 3.17 provide another workaround because we are rolling our own then, see https://github.com/kokkos/kokkos/blob/7ff4042fd2dc1d1fe6ae280179fc693ad6dac6ac/cmake/Modules/FindTPLCUDA.cmake#L14-L39.